nodes/config/ui_base.js

```diff
@@ -172,11 +194,13 @@ module.exports = function (RED) {
         }, 300)
 
         // Make sure we clean up after ourselves
-        node.on('close', async (done) => {
-            // clear event handlers on base node
-            ui.ioServer?.off('connection', onConnection)
```

Comment:
why has this been removed? We want to remove the `ioServer` event handlers when a widget is deleted, which, if I can see correctly, you're no longer doing?

Reply:
as discussed - resolved in [e0e33b6](https://github.com/flowforge/flowforge-nr-dashboard/pull/97/commits/e0e33b68b474778909a503221a72150159a2e2f9)